WordPress/Classes/Plugins/Views/PluginDetailsView.swift

```diff
@@ -18,11 +30,38 @@ struct PluginDetailsView: View {
 
     @Environment(\.dismiss) var dismiss
 
+    var wpOrgURL: URL? {
+        URL(string: "https://wordpress.org/plugins/\(slug.slug)/")
+    }
+
+    var actionButton: ActionButton {
+        if let plugin = viewModel.installed {
```

Comment:
Create the buttons here directly? What's the point of `ActionButton`?

Reply:
I thought it'd be more apparent to see what buttons can be displayed on the screen (the enum cases). Using if-else statements may not be as clear.